src/app/home/home.page.html

```diff
@@ -18,5 +18,6 @@
     <ion-button (click)="onHomeBtnClick()" fill="clear"><ion-icon name="albums"></ion-icon></ion-button>
     <ion-button (click)="onProfileBtnClick()" fill="clear"><ion-icon name="person"></ion-icon></ion-button>
     <ion-button (click)="onSettingsBtnClick()" fill="clear"><ion-icon name="Edit Your Attributes"></ion-icon></ion-button>
+    <ion-button (click)="onGoToNotificationBtnClick()" fill="clear"><ion-icon name="notifications"></ion-icon></ion-button>
```

Comment:
Eventually, we will update the footer buttons, but for now I'm not sure what all should go in there. Instead, you should add a button like the one on line 12 to launch the Notifications page.

Reply:
Quick fix: Just committed the requested changes.

Let me know if anything else needs to be changed.